WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewDataSource.swift

```diff
@@ -277,13 +279,31 @@ private extension ProductFormTableViewDataSource {
         cell.hideSeparator()
     }
 
+    func configurePromoteWithBlaze(cell: UITableViewCell) {
+        guard let cell = cell as? LeftImageTableViewCell else {
+            fatalError()
```

Comment:
Nit: we should set a message to make it easier to understand the error in Sentry if it ever happens.

```suggestion
            fatalError("Unexpected table view cell for the promote with Blaze cell")
```